docs/model-dev-guide/apis-howto/api-core-ug.rst

```diff
@@ -1,214 +1,299 @@
-      :language: python
-      :pyobject: save_state
+To enable pausing an experiment, enable preemption:
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/2_checkpoints.py
-      :language: python
-      :pyobject: load_state
+.. literalinclude:: ../../../examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py
+   :language: python
+   :start-after: # Docs snippet start: enable preemption
+   :end-before: # Docs snippet end: enable preemption
+   :dedent:
 
-#. In your ``if __name__ == "__main__"`` block, use the ClusterInfo API to gather additional
-   information about the task running on the cluster, specifically a checkpoint to load from and the
-   trial ID, which you also pass to ``main()``.
+Define a load_state function for restarting model training from existing checkpoint:
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/2_checkpoints.py
-      :language: python
-      :start-at: info = det.get_cluster_info()
+.. literalinclude:: ../../../examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py
+   :language: python
+   :start-after: # Docs snippet start: define load state to restart
+   :end-before: # Docs snippet end: define load state to restart
+   :dedent:
 
-   It is recommended that you always follow this pattern of extracting values from the ClusterInfo
-   API and passing the values to lower layers of your code, instead of accessing the ClusterInfo API
-   directly in the lower layers. In this way the lower layer can be written to run on or off of the
-   Determined cluster.
+If checkpoint exists, load it and assign it to model state prior to resuming training:
 
-#. Within ``main()``, add logic to continue from a checkpoint, when a checkpoint is provided:
+.. literalinclude:: ../../../examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py
+   :language: python
+   :start-after: # Docs snippet start: if checkpoint assign to model state
+   :end-before: # Docs snippet end: if checkpoint assign to model state
+   :dedent:
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/2_checkpoints.py
-      :language: python
-      :start-at: def main
-      :end-at: for batch in range(starting_batch, 100)
+**Enable Continuing the Trial**
 
-#. You can checkpoint your model as frequently as you like. For this exercise, save a checkpoint
-   after each training report, and check for a preemption signal after each checkpoint:
+To enable continuing the trial after the experiment completes, save the trial ID. One way to do this
+is to load the checkpoint and save the checkpoint in a file in the checkpoint directory.
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/2_checkpoints.py
-      :language: yaml
-      :dedent:
-      :start-at: if steps_completed % 10 == 0
-      :end-before: core_context.train.report_validation_metrics
+Open the `checkpoint.pt` file in binary mode and compare `ckpt_trial_id` with the current
```

Comment:
I think `checkpoint.pt` and similar on this line were intended to have two backticks surrounding them to get code style. The single backticks lead to italicizing

Reply:
good find